colibre/scripts/deadtime_evolution.py

```diff
@@ -0,0 +1,90 @@
+"""
+Plots dead time fraction v.s. step size
```

Comment:
What you probably want to write here is, e.g.,

_Plots dead time fraction v.s. cosmic scale factor_